src/ui/view/governance/details/details.tsx

```diff
@@ -43,80 +45,135 @@ const iconMapping: Record<string, string> = {
   Users: 'users'
 }
 
+// eslint-disable-next-line max-lines-per-function
 const Paragraph: FC<ParagraphProps> = ({ paragraph, theme }) => {
   const [isTextTooLong, setIsTextTooLong] = useState<boolean>(false)
+  const { t } = useTranslation('common')
   const paragraphRef = useCallback((node: HTMLParagraphElement | null) => {
     if (node !== null) {
       setIsTextTooLong(node.offsetHeight < node.scrollHeight)
     }
   }, [])
 
-  return (
-    <div className="okp4-dataverse-portal-governance-details-paragraph-container">
-      <div className="okp4-dataverse-portal-governance-details-paragraph">
-        <Icon name={`${iconMapping[paragraph.title] || 'description'}-${theme}` as IconName} />
-        <h3 className="okp4-dataverse-portal-governance-details-paragraph-title">
-          {paragraph.title}
-        </h3>
-        <p
-          className="okp4-dataverse-portal-governance-details-description paragraph"
-          ref={paragraphRef}
-        >
-          {paragraph.description}
-        </p>
-        {isTextTooLong && (
-          <div className="okp4-dataverse-portal-governance-details-button">
-            <Button disabled label="See more" variant="outlined-tertiary" />
-          </div>
+  const paragraphDescription = useMemo(() => {
+    const descriptionWords = paragraph.description.slice().split(' ')
+    return (
+      <div
+        className="okp4-dataverse-portal-governance-details-description paragraph"
+        ref={paragraphRef}
+      >
+        {descriptionWords.map((word, index) =>
+          isDID(word) ? (
+            word.length > 22 ? (
+              <div
+                className="okp4-dataverse-portal-governance-details-description-paragraph-did"
+                key={index}
+              >
+                <span>{[word.slice(0, 16), '...', word.slice(-4)].join('')}</span>
```

Comment:
We probably can create a utility function that group it with [this one](https://github.com/okp4/dataverse-portal/blob/b307d781f9c2f22aad50a88d554f0525f2c51ee9/src/ui/component/toolbar/toolbar.tsx#L252).